### DISCUSSION TITLE
Automatically disable "Prime Subscription and Loot Reminder" and others

### DISCUSSION THREAD

**Discussion by sillyElvy** [night/betterttv#5883] [Ideas]
It is pretty annoying when those things (IDK what they are called) show up whenever you hover over the player, so it would be cool if there was a setting to hide them automatically if that is possible.

If this already exists please point me to the location because I could not find it. (The "prime promotions" setting does not hide the "free loot" and others on the player)

**Comment by night** [comment]
This should exist already.. Have you tried changing the `Player Extensions` setting?

**Reply by sillyElvy** [reply]
That was it, thanks!
